torch_dftd/functions/dftd3.py

```diff
@@ -262,29 +265,40 @@ def edisp(
         # r_abc = r[within_cutoff]
         # r2_abc = r2[within_cutoff]
         edge_index_abc = edge_index[:, within_cutoff]
+        batch_edge_abc = None if batch_edge is None else batch_edge[within_cutoff]
         # c6_abc = c6[within_cutoff]
         shift_abc = None if shift is None else shift[within_cutoff]
 
         n_atoms = Z.shape[0]
         if not bidirectional:
             # (2, n_edges) -> (2, n_edges * 2)
-            edge_index_abc = torch.cat([edge_index_abc, edge_index_abc.flip(dims=[1])], dim=1)
+            edge_index_abc = torch.cat([edge_index_abc, edge_index_abc.flip(dims=[0])], dim=1)
```

Comment:
Is this tested?

Reply:
I have added a test for `bidirectional=True` and `abc=True`.
I confirmed that the tests fail with the previous `dims=[1]` version.